src/components/Select/props.ts

```diff
@@ -110,4 +118,21 @@ export type SelectProps<ValueType = unknown> = BaseInputProps & {
    * Customize the rendering dropdown options.
    */
   optionRender?: OptionRender<ValueType>
+
+  /**
+   * If true, filter options by input, if function, filter options against it.
+   * The function will receive two arguments, inputValue and option, if the function
+   * returns true, the option will be included in the filtered set; otherwise, it will be excluded.
+   */
+  filterOption?: FilterOption<ValueType>
+
+  /**
+   * The prop name to filter options. Default is 'value'.
+   */
+  optionFilterProp?: string
+
+  /**
+   * Customize the rendering of the dropdown.
+   */
+  dropdownRender?: (menu: ReactNode) => ReactElement
```

Comment:
Would you be able to add showSearch props?

Reply:
Why? It is useful to have it separated to onSearch and filter props?